### DISCUSSION TITLE
Is there a way to manually export an ebiten.Image to a png or other image format

### DISCUSSION THREAD

**Discussion by ThePuzzledDev** [hajimehoshi/ebiten#2854] [Q&A] [answered]
I have an image that I manually generated which isn't properly rendering. I would like to export to a pong or other image format in order to help debug and understand what's wrong with it. Is there a function that allows you to do this. Otherwise I'll submit a feature request.

**Comment by hajimehoshi** [comment]
I think you are talking about your game (https://github.com/hajimehoshi/ebiten/issues/2852), and the background was blue on browsers while gray on desktops. My guess is that you are using wrong colors as premultiplied alpha colors. See https://github.com/tinne26/kage-desk/blob/main/docs/tutorials/premult.md

**Comment by hajimehoshi** [comment, marked as answer]
>  I would like to export to a pong or other image format in order to help debug and understand what's wrong with it. 

Use `EBITENGINE_SCREENSHOT_KEY` environment variable (https://pkg.go.dev/github.com/hajimehoshi/ebiten/v2#hdr-Environment_variables)
